<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeEditDialog</class>
 <widget class="QDialog" name="RecipeEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Льготный рецепт</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="2">
    <widget class="CDloFormularyComboBox" name="cmbDrug"/>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbtnVkYes">
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnVkNo">
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbtn50proc">
        <property name="text">
         <string>50%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn100proc">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Срок действия</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Наличие протокола ВК</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Количество единиц</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Препарат</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Источник финансирования</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="spbNumPerDay"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Процент оплаты</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>МКБ</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QComboBox" name="cmbTerm">
     <item>
      <property name="text">
       <string>5 дней</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 дней</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 дней</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90 дней</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Длительность приёма</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Приёмов в день</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="CICDCodeEditEx" name="cmbMKB"/>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="edtDosage"/>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="spbDuration">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Код категории граждан</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="spbQnt">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Дозировка</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QComboBox" name="cmbStatus">
     <item>
      <property name="text">
       <string>действителен</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>недействителен</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>испорчен</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="cmbSocCodes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QLineEdit" name="edtSigna"/>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>дней</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Порядок приёма</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="edtDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>№ карты беременной</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtPregCard">
     <property name="inputMask">
      <string>00000000000000000000; </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>(если заполнено, выводится как № карты больного)</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Выводить на печать</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbtnPrintMNN">
        <property name="text">
         <string>МНН</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnPrintTradeName">
        <property name="text">
         <string>Торговое наименование</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEditEx</class>
   <extends>QComboBox</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CDloFormularyComboBox</class>
   <extends>QComboBox</extends>
   <header>library/Recipe/DloFormularyComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbSocCodes</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>cmbMKB</tabstop>
  <tabstop>cmbDrug</tabstop>
  <tabstop>edtDosage</tabstop>
  <tabstop>spbQnt</tabstop>
  <tabstop>spbNumPerDay</tabstop>
  <tabstop>spbDuration</tabstop>
  <tabstop>edtSigna</tabstop>
  <tabstop>cmbTerm</tabstop>
  <tabstop>cmbStatus</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecipeEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecipeEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
